spec/files/downloader_spec.ts

```diff
@@ -82,10 +82,23 @@ describe('downloader', () => {
     let fileUrl =
         'https://selenium-release.storage.googleapis.com/3.0/selenium-server-standalone-3.0.0.jar';
     let fileName = 'foobar.jar';
-    let outputDir = Config.getSeleniumDir();
+    let outputDir = path.resolve('selenium_test');
     let actualContentLength = 22138949;
     let contentLength: number;
 
+    beforeEach(() => {
+      outputDir = path.resolve('selenium_test');
```

Comment:
Good catch on `process.cwd()` being redundant, but that actually wasn't what I was nit picking about :laughing:

Why do you need to set `outputDir` in the `beforeEach`?  You set it back on line 85, and I don't see it changing (or needing to change).